packages/analytics-types/src/payload.ts

```diff
@@ -8,4 +8,5 @@ export interface Payload {
   api_key: string;
   events: readonly Event[];
   options?: PayloadOptions;
+  client_upload_time?: string;
```

Comment:
Is this different from `time`?

Reply:
yes, they are different. `time` in SDK is [`client_event_time` in background](https://github.com/amplitude/amplitude/blob/90b19342d7d991bb4854102ad85fda22b299a3b3/event/converters/http_event.py#L254-L258).

Reply:
Same question, seems similar as `time` field except that this is ISO format.

Reply:
Please see more background [here](https://amplitude.slack.com/archives/C02E1H87E2C/p1697495952472679)

Reply:
Closed it as getting approved from Kevin from mid week oncall sync